src/Microsoft.AspNet.Diagnostics/Views/StackFrame.cs

```diff
@@ -32,19 +33,19 @@ public class StackFrame
         public int PreContextLine { get; set; }
 
         /// <summary>
-        /// 
+        /// Lines of code before the actual error line(s)
         /// </summary>
-        public IEnumerable<string> PreContextCode { get; set; }
+        public IEnumerable<string> PreContextCode { get; set; } = Enumerable.Empty<string>();
```

Comment:
Why add these? Very odd to declare enumerables as enumerable.empty in poco classes.